src/mon/OSDMonitor.cc

```diff
@@ -1833,11 +1833,11 @@ bool OSDMonitor::can_mark_out(int i)
   float in_ratio = (float)in / (float)num_osds;
   if (in_ratio < g_conf->mon_osd_min_in_ratio) {
     if (i >= 0)
-      dout(5) << "can_mark_down current in_ratio " << in_ratio << " < min "
+      dout(5) << "can_mark_out current in_ratio " << in_ratio << " < min "
```

Comment:
The problem is that we manually output the function name everywhere within this file.
So I perfer to keep pace with others.

But I'll do it if you insist.